src/components/elements/input/GenericSelect.tsx

```diff
@@ -65,6 +67,7 @@ const GenericSelect = <
             ...inputProps.inputProps,
           }}
           disabled={rest.disabled}
+          aria-label={ariaLabel}
```

Comment:
Ah thanks @martha , I also fixed this on a branch I'm working on https://github.com/greenriver/hmis-frontend/pull/676/files but I passed it through a separate prop so that the `aria-label` gets applied to the actual `input`, not the surrounding div. Aria label should go on interactive elements. Can you update to pass it through input props?